src/main/java/pl/winiarczyk/lesson4/homework/TextFile/InvertedCharacters.java

```diff
@@ -0,0 +1,17 @@
+package pl.winiarczyk.lesson4.homework.TextFile;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class InvertedCharacters {
+    public static void main(String[] args) throws IOException {
+
+        String text = Files.readString(Path.of("C:\\Users\\Lenovo\\Java23\\data.txt"));
```

Comment:
https://github.com/Sticky5/Java23/pull/31#discussion_r707019734

Reply:
ale tu dam Ci approve (w poprzednim zadaniu odrzuciłem), ale popraw to proszę tak czy siak ;)